Tests/Pages/User/ViewTest.php

```diff
@@ -8,16 +8,6 @@
     class ViewTest extends \Tests\KnownTestCase
     {
 
-        function testPagePermalink()
```

Comment:
Why remove test? 

If the test is failing it shouldn't be removed, the root cause needs to be fixed.

Reply:
It was a test I added and then quickly removed because we don't have the capability right now to do what I needed it to. I'll work on the test infra separately and then bulk out the tests.